src-tauri/src/app_config.rs

```diff
@@ -68,6 +72,8 @@ impl AppConfig {
             last_binaries_update_timestamp: SystemTime::now(),
             allow_analytics: true,
             anon_id: generate_password(20),
+            telemetry_collection: true,
```

Comment:
Can we merge allow_analytics and telemetry_collection fields? They do the same thing